### PR TITLE
Redo readme for pvb public

### PR DESCRIPTION
**Pull request by mloughran**
What it says on the tin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
-a day or date.
+### Local development
 
-#### `ConcreteSlot`
+The default configuration of `PRISON_VISITS_API` is http://localhost:3000/, so the least-friction development approach is to run this app on another port.
 
-The expression of a slot on a particular day: this is a unique time that can be
-booked.
+          pvb2 $ rails server
+    pvb-public $ rails server -p 4000
```
Comment (alan) [inline README.md]: Is it worth mentioning that depends also on the nomis api?

Comment (mloughran) [inline README.md]: No, that's an implementation detail of pvb2.

File: README.md [hunk 2]
```diff
@@ -239,10 +95,6 @@ Make sure the secret is at least 30 characters and all random, no regular words
 or you’ll be exposed to dictionary attacks. You can use `rake secret` to
 generate a secure secret key.
 
-#### `SESSION_SECRET_KEY`
-
-This is used to sign the session used by the Sidekiq admin interface.
-
 #### `SMTP_USERNAME`, `SMTP_PASSWORD`
```
Comment (alan) [inline README.md]: Do we need these? I don't think it talks to sendgrid the public app.

File: README.md (hunk 1 above)
Comment (mloughran) [inline README.md]: Besides, pvb2 will currently function just fine without nomis.

File: README.md (hunk 2 above)
Comment (mloughran) [inline README.md]: I thought the same, but in fact the sendgrid API calls are still in public. They can be removed when we move email validation to a pvb2 API call (as discussed), or better yet remove entirely (as also discussed!).

File: README.md (hunk 1 above)
Comment (alan) [inline README.md]: 👍 Maybe then add a link to the local development for pvb2? That way someone new will know that the information here might not be enough to get the whole stack running.

Comment (mloughran) [inline README.md]: I will do, when I've written the readme for PVB2. I was going to wait until we remove all the public stuff from it if that's ok.

File: README.md (hunk 2 above)
Comment (alan) [inline README.md]: I thought that it was already was dead code in public but I see it is being used by the visitors step model

File: README.md (hunk 1 above)
Comment (alan) [inline README.md]: 👍

**Pull request merged**